syftlib/publish.gradle

```diff
@@ -1,16 +1,29 @@
-apply plugin: 'com.novoda.bintray-release'
+apply plugin: "nebula.maven-publish"
+apply plugin: "nebula.javadoc-jar"
+apply plugin: "nebula.source-jar"
 
-publish {
+apply plugin: "nebula.nebula-bintray-publishing"
 
-    def groupProjectID = 'org.openmined'
-    def artifactProjectID = 'syftlib'
-    def publishVersionID = '0.0.1'
+Properties properties = new Properties()
+def propertiesFile = project.rootProject.file('local.properties')
+if (propertiesFile.exists()) {
+    properties.load(propertiesFile.newDataInputStream())
+}
 
-    userOrg = 'openmined'
-    repoName = 'KotlinSyft'
-    groupId = groupProjectID
-    artifactId = artifactProjectID
-    publishVersion = publishVersionID
-    desc = 'Android worker library'
-    website = 'https://github.com/OpenMined/KotlinSyft'
-}
\ No newline at end of file
+def bintrayUser = properties.getProperty('bintray.user')
+def bintrayKey = properties.getProperty('bintray.key')
+
+bintray {
```

Comment:
Awesone! Does the lib need tags to be added in the repo now? Additionally, if travis files need to be changed for knowing when to publish or use custom gradle build command?

Reply:
I'd go step by step. It already took me some time to figure out this first part :)
I think we need to set a first tag, yes. I will do that.
Then there is some other things to do if pushing from the command line with the branch name patterns but code-wise, it's ready.

Reply:
Afaik you would need to change the Travis files as well. I would prefer that also gets added here in this PR